src/gfx/shader/wgsl_mapping.hpp

```diff
@@ -59,16 +59,16 @@ inline String getFieldWGSLTypeName(const FieldType &type) {
   const char *baseType = nullptr;
   switch (type.baseType) {
   case ShaderFieldBaseType::UInt8:
-    baseType = "u8";
+    baseType = "u32";

```

Comment:
Why is the base 32??

Reply:
WebGPUdoesn't support anything useful on anything less than <u32/i32 in wgsl 
So when defining a field it's defined as u32 in the shader, but it might come from an uint8 vertex attribute in memory

References:
https://www.w3.org/TR/WGSL/#integer-types
https://www.w3.org/TR/webgpu/#enumdef-gpuvertexformat

Reply:
I see, I thought it's for the attributes. Ok, makes sense. They'll add support for fp16 at some point, but I think it's disabled by default unless you compile with that option.